src/app/pages/account/groups/group-form/group-form.component.ts

```diff
@@ -158,6 +169,81 @@ export class GroupFormComponent implements OnInit {
     });
   }
 
+  private togglePrivilegesForGroup(groupId: number): Observable<Privilege[]> {
+    const requests$: Observable<Privilege>[] = [];
+
+    if (this.selectedPrivilegeIds) {
+      const privileges = this.privilegesList
```

Comment:
Is this filter required? User should only be allowed to select valid privilege values when they are editing the form.

Reply:
@RehanY147  - yes, how without filter we'll know what was added and what was removed?
How to save data eventually?